userTags/README.md

```diff
@@ -0,0 +1,12 @@
+# User Tags
+Firestore collection model for user related tags & its metadata
+
+```
+{
+    userId: String,
+    tagId: String,

```

Comment:
can this be an array, because there can be multiple tags?

Reply:
i'm thinking of storing it as `"react": ""`

Reply:
why we would store empty strings inside the key?

Reply:
it can be 1 of 2 ways.
- create series of tags(tag1, tag2) as key & store values in it
- store value as key with empty string

Reply:
we will take approach as it was discussed in call to create a new doc for every user tag along with their id
e.g.
```
doc1: {
    userid: user123,
    tagid: fdhjewuih3
},
doc2: {
    userid: user123,
    tagid: fdwjk2
}
```